Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/IDE/Wing/constants.py

```diff
@@ -0,0 +1,73 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+# -------------------------------------------------------------------------
+"""Common constants for the dccsi wing pro ide integration
+
+:file: < DCCsi >/Tools/IDE/Wing/constants.py
+:Status: Prototype
+:Version: 0.0.1
+"""
+# -------------------------------------------------------------------------
+# standard imports
+import logging as _logging
+# -------------------------------------------------------------------------
+_MODULENAME = 'DCCsi.Tools.IDE.Wing.constants'
+_LOGGER = _logging.getLogger(_MODULENAME)
+_LOGGER.debug(f'Initializing: {_MODULENAME}')
+# -------------------------------------------------------------------------
+# dccsi imports here
+# wing is a x86 program, this pulls the constant for that (not a Path object)
+from DccScriptingInterface.constants import PATH_PROGRAMFILES_X86
+from DccScriptingInterface.Tools.IDE.Wing import PATH_DCCSI_TOOLS_IDE_WING
+# -------------------------------------------------------------------------
+# wing pro ide constants here
+# a setting to know the wing config exists
+ENVAR_DCCSI_CONFIG_IDE_WING = 'DCCSI_CONFIG_IDE_WING'
+
+# enavr to get/set the wing pro version
+ENVAR_DCCSI_WING_VERSION_MAJOR = 'DCCSI_WING_VERSION_MAJOR'
+
+# the default supported version of wing pro is 8
+SLUG_DCCSI_WING_VERSION_MAJOR = int(8)
+
+# str slug for the default wing type
+# in the future, add support for wing personal and maybe wing 101 versions
+SLUG_DCCSI_WING_TYPE = 'Wing Pro'
+
+# path string constructor, dccsi default WINGHOME location
+PATH_WINGHOME = (f'{PATH_PROGRAMFILES_X86}' +
+                f'\\{SLUG_DCCSI_WING_TYPE} {SLUG_DCCSI_WING_VERSION_MAJOR}')
+
+# wing native hook for it's home location, used in wingstub.py
+ENVAR_WINGHOME = 'WINGHOME'
+
+ENVAR_WINGHOME_BIN = 'WINGHOME_BIN'
+
+# path string constructor, wing home bin folder
+PATH_WINGHOME_BIN = f'{PATH_WINGHOME}\\bin'
+
+# envar to get/set the wing pro exe
+ENVAR_WING_EXE = 'WING_EXE'
+
+# path string constructor, wing home bin folder
+PATH_WINGHOME_BIN_EXE = f'{PATH_WINGHOME_BIN}\\wing.exe'
+
+# this is already available PATH_DCCSI_TOOLS_IDE_WING, imported above
```

Comment:
Can we remove this commented out code? Or is it valuable to explain where that comes from?